src/main/java/lotto/domain/WinLotto.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class WinLotto extends Lotto {
+    private final LottoNumber bonusNumber;
```

Comment:
Lotto 를 상속받으신 이유가 있으실까요? 😄 
```suggestion
public class WinLotto {
    private final LottoNumber bonusNumber;
    private final Lotto winLotto;
```

Reply:
제 생각에 Lotto는 6숫자를  지니는 한게임의 단위로 생각을 했는데, 보너스 번호의 경우에는 당첨번호에만 해당하는 특수케이스 인것 같아서 상속으로 이용해서 구현했습니다.

이 부분에 대해서는 어떻게 생각하시는지 궁금합니다!

Reply:
[이팩팀브 자바에서는 상속보다는 컴포지션을 활용하라](https://ckddn9496.tistory.com/92) 라고 이야기 하는데요
관련해서 한번 고민해보시면 좋을 것 같아요! 😄

Reply:
조언해주신대로 WinLotto를 컴포지션을 이용해서 수정하였습니다. 

이 부분에 대해서는 상속과 컴포지션에 대해서 의문이 살짝 남아있는데요.
제가 생각하는 상속과 컴포지션의 차이는 
1. 인스턴스 변수로 객체를 의존하는지, extends로 의존하는지
2. 다형성 이용이 가능한지
3. 구현의 변경 여부( override)
가 크게 당장은 떠오르네요.
인터넷으로 찾아보니 다음과 같은 차이점이 존재하네요[inheritance vs composition](https://www.tutorialspoint.com/difference-between-inheritance-and-composition-in-java)

저는 이부분에 대해서 아직은 걸음마 단계라서 사실 컴포지션과 상속이 나올때마다 궁금했던 부분이기도 한데요.
이슬님이 생각하시는 컴포지션의 이점과 상속과 컴포지션은 언제 활용하는게 좋을지 판단하시는 노하우를 공유해주시면 정말 감사하겠습니다:)

링크에서 언급한것처럼 is-a 관계이면 상속을 이용하고, has-a 관계이면 composition을 이용하는 방식도 잊고있었던 좋은 방법인것 같습니다.